spec/unit/util_spec.rb

```diff
@@ -259,33 +259,34 @@ def withenv_utf8(&block)
       end
     end
 
-    describe "when using platform :posix" do
-      %w[/ /foo /foo/../bar //foo //Server/Foo/Bar //?/C:/foo/bar /\Server/Foo /foo//bar/baz].each do |path|
-        it "should return true for #{path}" do
-          expect(Puppet::Util).to be_absolute_path(path, :posix)
-        end
-      end
-
-      %w[. ./foo \foo C:/foo \\Server\Foo\Bar \\?\C:\foo\bar \/?/foo\bar \/Server/foo foo//bar/baz].each do |path|
-        it "should return false for #{path}" do
-          expect(Puppet::Util).not_to be_absolute_path(path, :posix)
-        end
-      end
-    end
-
-    describe "when using platform :windows" do
-      %w[C:/foo C:\foo \\\\Server\Foo\Bar \\\\?\C:\foo\bar //Server/Foo/Bar //?/C:/foo/bar /\?\C:/foo\bar \/Server\Foo/Bar c:/foo//bar//baz].each do |path|
-        it "should return true for #{path}" do
-          expect(Puppet::Util).to be_absolute_path(path, :windows)
-        end
-      end
-
-      %w[/ . ./foo \foo /foo /foo/../bar //foo C:foo/bar foo//bar/baz].each do |path|
-        it "should return false for #{path}" do
-          expect(Puppet::Util).not_to be_absolute_path(path, :windows)
-        end
-      end
-    end
+    # TODO: we wouldn't need these if we could find a way to fake out the OS
```

Comment:
The hope is that we don't need this anymore - but that relies on us being able to rework `file_serving/base`